packages/web/public/static/locales/en/post.json

```diff
@@ -39,5 +39,12 @@
     "body": "Are you sure you want to cancel editing this post? Unsaved progress will be lost."
   },
   "enterUrlPrompt": "Enter the URL of the link:",
-  "websitePatternError": "Invalid website url. e.g. domain.com or sub.domain.com"
+  "websitePatternError": "Invalid website url. e.g. domain.com or sub.domain.com",
+  "claps": {
+    "currentUserPronoun": "You",
+    "noUsersClapped": "0 users clapped for this post.",
```

Comment:
I think this could be a nice opportunity to further our fun and playful tone. Perhaps something like at least adding "...yet!" on the end, but I wonder if there's something even more creative 🧐 I'll have a think an happy to get your ideas if you have any! This is of course a super minor thing 😆

Reply:
Done